docs/tutorials/multimodal/multilingual_distillation.md

```diff
@@ -0,0 +1,185 @@
+# Knowledge Distillation in AutoMM for Text - in A Multilingual Problem
+:label:`sec_automm_distillation_multilingual`
+
+During the deployment of neural networks, limited resources always
+prevent us from using a larger models for better performance. 
```

Comment:
We can also mention that the pretrained foundation models are becoming increasingly larger these days. Reference to the Turing NLG figure in https://www.microsoft.com/en-us/research/blog/using-deepspeed-and-megatron-to-train-megatron-turing-nlg-530b-the-worlds-largest-and-most-powerful-generative-language-model/.

Model distillation is a technique for making these large models practical. We first train a teacher model and then distill the knowledge from the teacher model to a student model.